### PR TITLE
Update HealthCheckTests.cs

### PR DESCRIPTION
**Pull request by darkblue-ac**
## Purpose

* ...

## Does this introduce a breaking change?

```
[ ] Yes
[ ] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Get the code

```
git clone [repo-address]
cd [repo-name]
git checkout [branch-name]
npm install
```

* Test the code

```
```

## What to Check
Verify that the following are valid
* ...

### CODE REVIEW

**Review by rliegmann — DISMISSED** [review]
gut

File: apis/poi/tests/UnitTests/HealthCheckTests.cs [hunk 1]
```diff
@@ -10,7 +10,7 @@ public class HealthCheckUnitTests
         public void HealthCheckTestModel()
         {
             Assert.Equal("POI Service Healthcheck", new Healthcheck().Message);
-            Assert.Equal("Healthy", new Healthcheck().Status);
+            Assert.Equal("Healthy2", new Healthcheck().Status);
```
Comment (Schwembe) [inline apis/poi/tests/UnitTests/HealthCheckTests.cs:14]: gut

**Review by Schwembe — APPROVED** [review]

**Comment by kleemae** [comment]
ist gut

**Review by rliegmann — APPROVED** [review]
subba

**Review by fredylally — APPROVED** [review]

**Pull request closed without merging**